src/aspire/classification/rir_class2d.py

```diff
@@ -148,6 +155,13 @@ def __init__(
             )
         self._bispectrum = bispectrum_implementations[bispectrum_implementation]
 
+        # Setup class selection
+        if selector is None:
+            selector = RandomClassSelector(seed=self.seed)
```

Comment:
This will change the Gallery tutorial. Not sure if that matters, but should we be explicit about our choice of `selector` in that tutorial?